src/components/info/macros.rs

```diff
@@ -14,3 +13,5 @@ macro_rules! generate_algo_list_for_yew {
             .collect::<Vec<yew::virtual_dom::VNode>>()
     }};
 }
+
+pub(crate) use generate_algo_list_for_yew;
```

Comment:
the current project is not a library. So, there is no sense to make it a `pub(crate)`